packages/project-disaster-trail/src/components/atoms/Orb.js

```diff
@@ -102,7 +102,7 @@ export default class Orb extends PureComponent {
         this.setState({ isComplete: true });
         clearInterval(timer);
       } else if (percent < 100 && isActive) {
-        const update = percent + 10;
+        const update = percent + 20;
```

Comment:
I'm actually likely going to remove this whole function. RadialGauge doesn't need it